manifests/a/Apache/Maven/3/3.9.9/Apache.Maven.installer.yaml

```diff
@@ -0,0 +1,20 @@
+# Created with YamlCreate.ps1 v2.4.1 $debug=NVS1.CRLF.7-4-6.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Apache.Maven
+PackageVersion: 3.9.9
+InstallerLocale: en-US
+InstallerType: zip
+Commands:
+- mvn
+ReleaseDate: 2024-08-17
+Installers:
+- Architecture: neutral
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: apache-maven-3.9.9/bin/mvn.cmd
```

Comment:
CMD files are not allowed by policy

Reply:
@trenly maven publish officially mvn.cmd for windows, How to can I do?